### PR TITLE
Fixes an instance where chat would block character naming

### PR DESCRIPTION
**Pull request by MaxIsJoe**
CL: [Fix] Fixes a long-standing issue where attempting to edit a character's name in the character UI would cause the chat UI to popup while typing.

### CODE REVIEW

**Review by Bod9001 — DISMISSED** [review]

File: UnityProject/Assets/Scripts/UI/Systems/Character/CharacterSettings.cs [hunk 1]
```diff
@@ -79,6 +81,7 @@ public void ShowCharacterEditor(CharacterSheet character)
 			characterSelector.SetActive(false);
 			characterEditor.SetActive(true);
 			characterEditorScript.LoadCharacter(character);
+			UIManager.Instance.isInputFocus = true;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/UI/Systems/Character/CharacterSettings.cs:89]: Do you remember to unfocused it?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/UI/Systems/Character/CharacterSettings.cs:89]: it might be worth putting on  disable just to make sure

**Comment by corp-0** [comment]
Since you are there, can you remove the name validation? Just let people call the characters whatever they want, for now. 

We can find strategies to validate names  accordingly to the character status later.

File: UnityProject/Assets/Scripts/UI/Systems/Character/CharacterSettings.cs (hunk 1 above)
Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Systems/Character/CharacterSettings.cs:89]: pushed a change where it disables the focus when the entire window disappears

**Comment by MaxIsJoe** [comment]
> Since you are there, can you remove the name validation? Just let people call the characters whatever they want, for now.
> 
> We can find strategies to validate names accordingly to the character status later.

I'll do it in another PR because Unity is being a bitch right now and I can't do anything.

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**